Protobuf/dolittle/interaction/events.relativity/claim.proto

```diff
@@ -7,7 +7,7 @@ syntax = "proto3";
 
 package dolittle.events.relativity;
 
-option csharp_namespace = "Dolittle.Runtime.Events.Relativity.Protobuf";
+option csharp_namespace = "Dolittle.Runtime.Grpc.Interaction.Protobuf";
```

Comment:
Same as earlier - drop .Protobuf from the namespace (I Know, this ripples through - sorry about that)